diff --git a/index.bun.js b/index.bun.js
index 815856bd386617571e24bda7b45da694c731cd04..db756763d09503666c4c1d096770aff853bc0269 100644
--- a/index.bun.js
+++ b/index.bun.js
@@ -9672,10 +9672,10 @@ class MarkdownRenderable extends Renderable {
   getListItemInputs(token, id) {
     const items = token.items ?? [];
     const start = token.start === "" || token.start === undefined || token.start === null ? 1 : Number(token.start);
-    const markerWidth = Math.max(1, ...items.map((_2, index) => (token.ordered ? `${start + index}.` : "-").length));
+    const markerWidth = Math.max(1, ...items.map((_2, index) => (token.ordered ? `${start + index}.` : "•").length));
     return items.map((item, index) => ({
       item,
-      marker: token.ordered ? `${start + index}.` : "-",
+      marker: token.ordered ? `${start + index}.` : "•",
       markerWidth,
       id: `${id}-item-${index}`
     }));
diff --git a/index.node.js b/index.node.js
index 0ddf03bf666a05f50c66bef32af65b1403ed0192..aa4e2d280d8b79e445356f76b2b8f89dce84242a 100644
--- a/index.node.js
+++ b/index.node.js
@@ -9671,10 +9671,10 @@ class MarkdownRenderable extends Renderable {
   getListItemInputs(token, id) {
     const items = token.items ?? [];
     const start = token.start === "" || token.start === undefined || token.start === null ? 1 : Number(token.start);
-    const markerWidth = Math.max(1, ...items.map((_2, index) => (token.ordered ? `${start + index}.` : "-").length));
+    const markerWidth = Math.max(1, ...items.map((_2, index) => (token.ordered ? `${start + index}.` : "•").length));
     return items.map((item, index) => ({
       item,
-      marker: token.ordered ? `${start + index}.` : "-",
+      marker: token.ordered ? `${start + index}.` : "•",
       markerWidth,
       id: `${id}-item-${index}`
     }));
